cmd/banner.go

```diff
@@ -90,13 +90,14 @@ var bannerCmd = &cobra.Command{
 					}
 				}
 				if i == 0 {
-					fmt.Println("Waiting for Supervisor to startup...")
+					fmt.Println("Waiting for Supervisor to start-up...")
 				}
 				time.Sleep(1 * time.Second)
 			}
+			fmt.Println("Supervisor is ready!\n\n")
```

Comment:
Maybe it is also not clear what is meant by Supervisor :thinking:  How about:

```suggestion
			fmt.Println("System is ready!")
			fmt.Println()
```

We could also go one step further and tell the user to use the browser or app. :thinking: